### DISCUSSION TITLE
OA folder set up in Synology NAS?

### DISCUSSION THREAD

**Discussion by Audiophile22** [openaudible/openaudible#894] [Q&A] [answered]
I would like to mount a folder for download and conversion for OA on my Synology Nas. There are a few folders in the Open Audible Library. Do I need to duplicate the file paths from my desktop MAC or is there an easier way to set it up? Thank you in advance.

**Comment by openaudible** [comment, marked as answer]
You can change the library location to a mounted NAS share. But if you're OK on extra space (and extra redundancy) I just set the web page output location to the NAS, which copies HTML/thumbnails, books to a central directory.  That is the easiest way without having to move files around.

**Reply by Audiophile22** [reply]
Thank you very much!

**Comment by oboe45** [comment]
That is exactly how mine is set up on Synolgy NAS 920 plus.  I use one
folder to accommodate dowloads from Audible and a second folder tobe the
Web file - works fine for me.

**Reply by Audiophile22** [reply]
Thank you very much!
